dask_cuda/benchmarks/utils.py

```diff
@@ -202,6 +212,8 @@ def get_cluster_options(args):
             worker_options["enable_infiniband"] = ""
         if args.enable_rdmacm:
             worker_options["enable_rdmacm"] = ""
+        if args.device_memory_limit:
+            worker_options["device_memory_limit"] = ""
```

Comment:
Now it's missing the value to be assigned, which I believe should be `args.device_memory_limit`.